README.md

```diff
@@ -15,13 +16,7 @@ $ npm install --save @farfetch/react-carousel
 import { Carousel } from '@farfetch/react-carousel';
 ```
 
-We also include css styles out-of-the-box, to use them just import the css by doing the following:
-
-```js
-import '@farfetch/react-carousel/styles.css'
-```
-
-If you want to style the carousel by yourself, ignore this step.
+We also include base components that only possess logic, that way we will allow a blank slate in terms of styling.
```

Comment:
Perhaps we should have a note about needing emotion, otherwise it wouldn't be visible anywhere and people could think it's css for example.

Reply:
Added a paragraph indicating we use Emotion for styling